packages/connect/extension-resolver.cjs

```diff
@@ -0,0 +1,29 @@
+let defaultResolver;
+
+function requireDefaultResolver() {
+  if (!defaultResolver) {
+    try {
+      defaultResolver = import (`jest-resolve/build/defaultResolver`);
+    } catch (error) {
+      defaultResolver = import (`jest-resolve/build/default_resolver`);
+    }
+  }
+
+  return defaultResolver;
+}
+
+
+module.exports = (request, options) => {
+  const {basedir, defaultResolver, extensions} = options;
+
+  if (!defaultResolver) {
+    defaultResolver = requireDefaultResolver();
+  }
+
+  try {
+    return defaultResolver(request, options);
+  } catch (e) {
+    console.log(`${e} - Temporary replace extension '.js' to '.ts' for tests to run.\n`)
+    return defaultResolver(request.replace(/\.js$/, '.ts'), options);
```

Comment:
Wrap this second call to `defaultResolver` in its own `try` / `catch` and log in the catch block saying something like "errored resolving a module after replacing js extension with ts" and then log the outer `e` (not the error in this catch block) before rethrowing the inner error (the error in this catch block).  That way both messages and stack traces will show up giving us as much context as possible to track down the problem